<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryInterface</class>
 <widget class="QDialog" name="queryInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211,211,211);
background-position: center center;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;
background:rgba(85,170,225,0);
	font-style: italic bold;
	font size:14px;</string>
   </property>
   <property name="text">
    <string>查询界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>751</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btn_query">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#00FFFF;

	border-style:outset;
	border-radius:10px;
</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>721</width>
       <height>301</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>装备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>是否归还</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出库时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>归还时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>原因</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出库天数</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="lineEdit_deliveryDate">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
     </property>
     <property name="placeholderText">
      <string>出库时间</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="lineEdit_queryTime">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000;
background-color: rgb(255, 255, 255);
	font-size:18px;
	border-style:outset;
	border-radius:10px;</string>
     </property>
     <property name="placeholderText">
      <string>入库时间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_time_query">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#00FFFF;

	border-style:outset;
	border-radius:10px;
</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>691</width>
       <height>321</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>装备编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>来源</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>入库时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>负责人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>有效期限</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_switch">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;

	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>切换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#00FFFF;

	border-style:outset;
	border-radius:10px;
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
